doc/manual_experimental.md

```diff
@@ -560,7 +560,8 @@ Not nil annotation
 `{.experimental: "notnil".}`.
 
 All types for which `nil` is a valid value can be annotated with the
-`not nil` annotation to exclude `nil` as a valid value:
+`not nil` annotation to exclude `nil` as a valid value. Note that only local
+symbols are checked.
```

Comment:
But globals should be checked too.

Reply:
The `trackProc` function only tracks local symbols. Similarly as this case, the strictdef analysis is applied to local symbols. I suppose it's hard to analyze global symbols, which might be used in functions arbitrarily. 

Globals are only checked by the direct illegal assignment if that is what you meant. I will rephrase it better. 

Or I might miss something?